app/src/main/java/io/gnosis/safe/ui/settings/owner/OwnerEnterNameFragment.kt

```diff
@@ -52,6 +55,7 @@ class OwnerEnterNameFragment : BaseViewBindingFragment<FragmentOwnerNameEnterBin
         with(binding) {
             newAddressBlockies.setAddress(ownerAddress)
             newAddressHex.text = ownerAddress.formatEthAddress(requireContext(), addMiddleLinebreak = false)
+            keyType.setImageResource(ownerType.getImageResForOwnerType())
```

Comment:
ownerType.getImageResForOwnerType sounds redundant. Probably could be shortened to ownerType.getImageRes. Or even better with a property: ownerType.imageRes

Reply:
It could be a property on OwnerViewData but shouldn't be a property on Owner.Type. So I kept it as an extension function for both. Also because we will need it in more places for the next PR.